nunit/MVP-Advance/SeleniumNUnit/Pages/ManageRequests.cs

```diff
@@ -0,0 +1,288 @@
+            wait(3);
+
+            //Look for the matching skill and click on it.
+            for (int i = 0; i < titles.Count(); i++)
+            {
+                if (titles[i].Text.Equals(skill))
+                {
+                    titles[i].Click();
+                    wait(3);
+
+                    //Enter message
+                    messageRequest.SendKeys(message);
+
+                    //Click on request button
+                    btnRequest.Click();
+                    wait(3);
+
+                    //Confirm
+                    btnYes.Click();
+                }
+                else
+                    Assert.Ignore("No skill found");
+            }
+        }
+        public void SwitchAccount(int accNumber)
+        {
+            //Initiate a Signin Object
+            SignIn signInObj = new SignIn();
+
+            //SignOut initial account to login a different account to make a request
+            btnSignOut.Click();
+            wait(3);
+            signInObj.LoginSteps(accNumber);
+        }
+
+        public void ClickSentRequests()
+        {
+            //Mousehover 
+            Actions action = new Actions(driver);
+            action.MoveToElement(manageRequests).Perform();
+            wait(3);
+
+            //Click "Sent Rquests" option
+            ddwnSentRequests.Click();
+            wait(3);
+        }
+
+        public void ClickReceivedRequests()
+        {
+            //Mousehover 
+            Actions action = new Actions(driver);
+            action.MoveToElement(manageRequests).Perform();
+            wait(3);
+
+            //Click "Received Rquests" option
+            ddwnReceivedRequests.Click();
+            wait(3);
+        }
+        public string GetSentRequests()
+        {
+            //Populate Excel and get request name
+            ExcelLib.PopulateInCollection(Base.ExcelPath, "ManageRequests");
+            string skill = ExcelLib.ReadData(2, "Title");
+
+            //Click "Sent Requests"
+            ClickSentRequests();
+
+            //Look for request and return request text
+            for (int i = 0; i < assertSentTitles.Count(); i++)
+            {
+                if (assertSentTitles[i].Text.Equals(skill))
+                {
+                    return assertSentTitles[i].Text;
+                }
+                else
+                {
+                    return "No matching request found.";
+                }
+            }
+            return "There's no request.";
+        }
+
+        public void WithdrawRequest()
+        {
+            //Populate Excel and get request name
+            ExcelLib.PopulateInCollection(Base.ExcelPath, "ManageRequests");
+            string skill = ExcelLib.ReadData(2, "Title");
+
+            //Click sent requests
+            ClickSentRequests();
+
+            //Get skill index in Sent Requests table
+            string index = "//div[@id='sent-request-section']//tbody/tr[" + GetSentSkillIndex(skill) + "]/td[8]/button";
```

Comment:
any reason this is not following the page object style?

Reply:
I explained above